keps/sig-cli/3895-kubectl-delete-interactivity/README.md

```diff
@@ -472,6 +472,7 @@ in back-to-back releases.
 #### Beta
 
 - Gather feedback from developers and surveys
+- KUBECTL_INTERACTIVE_DELETE environment variable is removed and interactive flag is enabled by default
```

Comment:
you mean available by default? It's not going to be the default behavior (ie, delete without `--interactive` is not going to become interactive)?

Reply:
@johnbelamaric thanks for review. You are right, `enable` is misleading. I change it to `available`.  Could you PTAL?.